deploy/kubernetes/csi-snapshotter/setup-csi-snapshotter.yaml

```diff
@@ -118,4 +118,4 @@ spec:
               mountPath: /csi
       volumes:
         - name: socket-dir
-          emptyDir:
+          emptyDir: {}
```

Comment:
Why is this needed?

Reply:
VSCode was complaining about it (`Incorrect type. Expected "object".`) when I went through the files and I must have impulsively... "fixed" it. 

To be on the safe side, I just ran kubeval on it. It doesn't mind either version (with or without empty object). I can remove that bit if you'd prefer to keep it as is to avoid complications in automation, sorry for not noticing myself when opening the PR.

Reply:
Let's keep it. I was just wondering because it seemed unrelated to the stated purpose of the PR.